src/Driver/IBMDB2/Statement.php

```diff
@@ -108,9 +107,9 @@ public function bindParam($param, &$variable, $type = ParameterType::STRING, $le
      */
     private function bind($position, &$variable, int $parameterType, int $dataType): void
     {
-        $this->bindParam[$position] =& $variable;
+        $this->parameters[$position] =& $variable;
 
-        if (! db2_bind_param($this->stmt, $position, 'variable', $parameterType, $dataType)) {
+        if (! db2_bind_param($this->stmt, $position, '', $parameterType, $dataType)) {
```

Comment:
The value of the 3rd argument is irrelevant here since it only matters if the variable with the given name is available in the scope from which `db2_execute()` is invoked, not this scope.

Since in our case `db2_bind_param()` and `db2_execute()` are invoked from different scopes, we have work around it by storing references to the bound parameters in a temporary map and pass them as 2nd argument to `db2_execute()`.